Okta/OktaAppAuth.swift

```diff
@@ -38,6 +38,10 @@ public func signOutOfOkta() -> SignOut {
     return SignOut()
 }
 
+public func authorize(withSessionToken sessionToken: String) -> Authorize {
```

Comment:
```suggestion
public func authenticate(withSessionToken sessionToken: String) -> Authorize {
```

Let's name this `authenticate`, not `authorize`. cc @robertjd @IldarAbdullin-okta

Reply:
Done.